tests/hardhat/Fork/comptrollerXVSAddressesUpgrade.ts

```diff
@@ -0,0 +1,110 @@
+import { smock } from "@defi-wonderland/smock";
+import { impersonateAccount } from "@nomicfoundation/hardhat-network-helpers";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import chai from "chai";
+import { ethers } from "hardhat";
+
+import { BEP20, Diamond, RewardFacet, SetterFacet } from "../../../typechain";
+import { forking } from "./utils";
+
+const { expect } = chai;
+chai.use(smock.matchers);
+
+const Owner = "0x939bd8d64c0a9583a7dcea9933f7b21697ab6396";
```

Comment:
Capitalized names usually refer to classes and types, I wouldn't expect it to be a string constant

```suggestion
const OWNER = "0x939bd8d64c0a9583a7dcea9933f7b21697ab6396";
```